README.md

```diff
@@ -53,14 +62,6 @@ Quick links:
     ├── requirements.txt        <- Requirements file for reproducing the analysis environment.
```

Comment:
- data - I don't think we will ever be able to store our data in the repo. That folder currently contains analysis of papers that tackle the problem of misogyny/hate speech and their data labeling process (not necessarily the datasets that we have).  It will also contain descriptions and stats about the current datasets (what I'm working on currently) and maybe instructions on how to request the datasets. 
- references - I don't know why this folder was created, but I don't think we need it, so I would remove it for now.
- tests - this folder will contains tests for the production data pipeline to make sure that it works as expected
- .flake8 - I don't know what this is, looks like it's used by the CI tools?
- .gitignore - this is a file that tells git which files to not add to repo when a developer makes a commit (automatically generated files, files that are too big, datasets etc.)
- .pre-commit-config.yaml - not sure maybe @malteserteresa knows what it is and if it serves any purpose right now?
- .pylintrc - same as above 
- DVcfile - something related to DVC
- LICENSE - looks automatically generated, we probably need one but I don't know much about this topic and under what LICENSE we are functioning
- mypy.ini - not sure what this is. @malteserteresa?

Reply:
Ah right, I missed the extension of the `gold_data_en.csv.dvc` file, I thought it was the actual dataset 😂
Ok will delete `references` and modify a couple of descriptions.